CONTRIBUTORS.md

```diff
@@ -0,0 +1,22 @@
+# Contributors
+
+## Project Lead
+* [Revathy Venugopal](https://github.com/Revathyvenugopal162)
+
+## Individual Contributors
+
+* [Alexander Kaszynski](https://github.com/akaszynski)
+* [Andy Grigg](https://github.com/Andy-Grigg)
+* [Dipin](https://github.com/dipinknair)
+* [Dominik Gresch](https://github.com/greschd)
+* [German](https://github.com/germa89)
+* [Guillem Barroso](https://github.com/GuillemBarroso)
+* [jleonatti](https://github.com/jleonatti)
```

Comment:
```suggestion
* [Jess Leonatti](https://github.com/jleonatti)
```

Reply:
Thanks @RobPasMue , i was double checking the names. This is so helpful.